src/metaprob/builtin_impl.clj

```diff
@@ -165,10 +165,10 @@
   ;; how to create a new binding in a namespace (a la def)???
   (let [sym (symbol name)
         r (ns-resolve the-ns sym)
-        r (if r r (binding [*ns* the-ns]
-                    (print (format "Assigning %s in %s" sym the-ns))
-                    (eval `(def ~sym))
-                    (ns-resolve the-ns sym)))]
+        r (or r (binding [*ns* the-ns]
```

Comment:
Using `or` in this way is perfectly natural for someone who knows Lisp or Scheme, but I think it's hackish